indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -59,6 +56,18 @@
             });
         });
 
+        $bulkMoveButton.on('click', function(evt) {
+            evt.preventDefault();
+            if ($(this).hasClass('disabled')) {
```

Comment:
Since you use it below too, let's put
```js
var $this = $(this);
```
at the top of the body of the function.